apmproxy/apmserver.go

```diff
@@ -96,10 +96,10 @@ func (c *Client) FlushAPMData(ctx context.Context) {
 	}
 
 	// Flush lambda data
+	c.logger.Debug("Flush in progress - Processing lambda data")
```

Comment:
[**For reviewers**] This log line led to a lot of noise during flush. Moved it so that only single entry is logged per flush.